api-contracts/openapi/openapi.yaml

```diff
@@ -34,6 +34,8 @@ paths:
     $ref: "./paths/v1/tasks/tasks.yaml#/listTasksByDAGIds"
   /api/v1/stable/tenants/{tenant}/workflow-runs:
     $ref: "./paths/v1/workflow-runs/workflow_run.yaml#/listWorkflowRuns"
+  /api/v2/stable/tenants/{tenant}/workflow-runs:
+    $ref: "./paths/v1/workflow-runs/workflow_run.yaml#/v2ListWorkflowRuns"
```

Comment:
naming here sucks - I wasn't sure what to call this so I called it `/v2`, but obviously that's problematic. open to ideas here!